config/config.yaml

```diff
@@ -64,6 +64,13 @@ components:
           items:
             $ref: '../lldp/lldp.yaml#/components/schemas/Lldp'
           x-field-uid: 9
+        egress_only_tracking:
+          description: |-
+            The list of structures containing Egress offset and it's length to be placed in the packet. 
```

Comment:
"Container for information on tracking desired based on certain offset and length of bits  on ports for received packets on specified receive ports. It enables the user to retrieve information based on number of unique packets received for each unique value in the bits being tracked from the beginning of the test ." --> Something of this nature might be more helpful to the user. The most appropriate description needs probably more detailed understanding but this is what I understand at a high level. [ But definitely nothing being placed in the packet ]